libkernelflinger/tpm2_security.c

```diff
@@ -564,6 +565,14 @@ EFI_STATUS tpm2_fuse_lock_owner(void)
 		return EFI_DEVICE_ERROR;
 	}
 
+	/* Check can read the bootloader NV index */
+	ret = read_tpm2_device_state(&state);
+	if (EFI_ERROR(ret)) {
+		// Read failed
```

Comment:
useless comment.  Of course if `ret` if an error we failed to read.  Try to avoid comments like this in general.  Plus the error message right after is a better way to comment the situation.